### ISSUE TITLE
Error: API rate limit exceeded

### ISSUE THREAD

**Issue by Qepanna** [Qepanna/goFlux#5]
This error is especially relevant for people updating the package on a regular basis (e.g. myself), especially when I am frequently pushing modifications to the package.

An error may occur when installing the package multiple times in a day (or a week) using the packages `remotes` and `devtools`.

```
try(detach("package:goFlux", unload = TRUE), silent = TRUE)
remotes::install_github("Qepanna/goFlux")
```

Here is the error:

```
message("Error: Failed to install 'goFlux' from GitHub:
  HTTP error 403.
  API rate limit exceeded for xxx.xxx.xxx.x (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

  Rate limit remaining: 0/60
  Rate limit reset at: 2023-10-16 09:08:07 UTC

  To increase your GitHub API rate limit
  - Use `usethis::create_github_token()` to create a Personal Access Token.
  - Use `usethis::edit_r_environ()` and add the token as `GITHUB_PAT`.")
```

I have created a troubleshoot protocol on the README and webpage documentation of the package. However, the problem is solved by creating a GitHub account, which I would rather avoid.

https://qepanna.quarto.pub/goflux/troubleshoot.html#error-api-rate-limit-exceeded

**Comment by Qepanna** [comment]
@Schuch666

**Comment by Schuch666** [comment]
Multiple installations from github can be made cloning the repository and installing via terminal (but this is only another workaround).

If you need to install to test installation, an automatic test or github actions can a useful implementation.

In case you are installing on your local machine is better to install from the local files (Ctr + Shift + b, if you are using RStudio).

Is there is a way to minimize the times you are installing from github?

**Comment by ibarraespinosa** [comment]
yeah, probably just mentioning explicitly in the documentation that the appropriate method to install and update is sync the repo would solve the issue.
